db/seeds.rb

```diff
@@ -1,7 +1,35 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+#Create users
+10.times do
+  User.create!(
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    password: 'helloworld'
+    )
+end
+users = User.all
+
+levels = ['public', 'private']
+
+100.times do
+  Wiki.create!(
+    subject: Faker::Lorem.sentence,
+    level: levels.sample,
+    body: Faker::Lorem.paragraph,
+    user: users.sample
+  )
+end
+wikis = Wiki.all
+
+50.times do
+  Collaboration.create!(
+    user: users.sample,
+    wiki: wikis.sample
+  )
+end
```

Comment:
Not a major problem, lines 25-30 could be creating duplicate collaborations. See previous comment on the multi-column unique index. The unique index would cause the db to prevent duplicate collaborations from being added.